### DISCUSSION TITLE
UE tested working

### DISCUSSION THREAD

**Discussion by improperaffo** [srsran/srsRAN_Project#219] [User Success Stories]
Dear all, regarding the COTS UE tested working, we would like to contribute to the [user reported section](https://docs.srsran.com/projects/project/en/latest/knowledge_base/source/cots_ues/source/index.html#user-reported-working-ues). We connected a OnePlus Nord CE 2 Lite 5G, a OnePlus Nord CE 3 Lite 5G and a Asus Zenfone 8. All of them were connected via a USRP b210 over a 20 MHz channel and 40 MHz channel on band n78 with PLMN 00101. Using open5gs core.

**Comment by andrepuschmann** [comment]
That's great @improperaffo - your input is much appreciated!

**Comment by robertfalkenberg** [comment]
@improperaffo Thanks for the input. Was the IMSI also starting with 00101, or did you configure a different IMSI that connected to 00101 PLMN via Roaming?

**Reply by improperaffo** [reply]
We had the IMSI set to 00101 both in the core settings and in the SIM cards, so no roaming.

**Reply by robertfalkenberg** [reply]
Much appreciated!

**Comment by improperaffo** [comment]
UPDATE:
The OnePlus Nord CE 2 Lite 5G seemed to have some stability issues out of the box, updating it to Android 13 seems to have solved them.
Also, we are using [Force 5G](https://play.google.com/store/apps/details?id=com.thephotoapps.force5g&hl=en_US&pli=1) to force them to use 5G only.

We also connected a Samsung S23, a Samsung A73 5G and an Acer Predator Connect D5 5G Dongle, same configuration of the network.

**Reply by pgawlowicz** [reply]
Thank you for your input.

**Comment by Eric-Forbes** [comment]
Pixel 7 Pro 
B200, External GPSO (LBE-1420), SIM: [sysmoISIM-SJA5-9FV](https://shop.sysmocom.de/sysmoISIM-SJA5-9FV-SIM-USIM-ISIM-Card-10-pack-with-ADM-keys-9FV-chip/sysmoISIM-SJA5-9FV-10p-adm)
B78, arfcn: 627340, PLMN: 001-01 

also working on a b210 w/o external 10MHz clock.
